noirup

```diff
@@ -48,7 +51,7 @@ main() {
     # Enter local repo and build
     say "installing from $NOIRUP_LOCAL_REPO"
     cd $NOIRUP_LOCAL_REPO
-    RUSTFLAGS="-C target-cpu=native" ensure cargo build --release # need 4 speed
+    RUSTFLAGS="-C target-cpu=native" ensure cargo build --release --features $FEATURES # need 4 speed
```

Comment:
```suggestion
    RUSTFLAGS="-C target-cpu=native" ensure cargo build --release --no-default-features --features $FEATURES # need 4 speed
```

If we're allowing people to specify feature flags then we should allow them to opt out of the default ones.